js/repl/compile.js

```diff
@@ -75,9 +79,9 @@ export default function compile(code: string, config: CompileConfig): Return {
       presets: config.presets,
       sourceMap: config.sourceMap,
     });
-
     compiled = transformed.code;
-
+    meta.compiled.size = new Blob([compiled], { type: "text/plain" }).size;
+    meta.raw.size = new Blob([code], { type: "text/plain" }).size;
```

Comment:
Kind of a nit, but we could also remove the `compiled` and `raw` wrapper objects here since we aren't bundling anything other than `size`.

If we inlined them more, we could also get rid of the unnecessary work being done by `_updateStatus` to massage their format before calling `setState` (and the `this.setState(result, setStateCallback)` would be sufficient)